api_contracts/contracts/award/AwardProfile.md

```diff
@@ -55,8 +55,32 @@ This endpoint returns a list of data that is associated with the award profile p
 + subaward_count: 430 (required, number)
 + total_subaward_amount: 35345353453 (required, number)
 + executive_details (required, Executive, fixed-type)
++ funding_obligated (required, number, nullable)
++ base_exercised_options (required, number, nullable)
+
+## IDVResponse (object)
++ type: `IDV_A` (required, string)
++ category: `idv` (required, string)
++ type_description: `BPA` (required, string)
++ piid: 34242 (required, number)
+    Award id
++ parent_award_piid: `1301` (required, string, nullable)
++ description: `ewraijwrw` (required, string, nullable)
+    Description of the first transaction for this award
++ awarding_agency (required, Agency, fixed-type)
++ funding_agency (required, Agency, fixed-type)
```

Comment:
Yes! In the overview section, we will be showing Awarding Agency.  This will also be the case for non-idv. 

This is because of a recommendation from our procurement SME.  Awarding Agencies are who spent the money out of their budget for the award.

Reply:
@AGuyNamedMarco So there won't be funding agencies?